x/msgfees/simulation/params.go

```diff
@@ -0,0 +1,39 @@
+package simulation
+
+// DONTCOVER
```

Comment:
What is this for?

Reply:
believe it is to ignore coverage, based on code in simulation pkg, since they are only used for simulation tests. (every module in our project has it)